test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -35,24 +35,24 @@ async function setup(t) {
   return pageObject;
 }
 
-fixture('Enroll YubiKey Authenticator');
+fixture('Enroll YubiKey Authenticator').meta('v3', true);
 test
   .requestHooks(logger, enrollMock)('enroll with YubiKey authenticator', async t => {
     const pageObject = await setup(t);
 
+    await t.expect(pageObject.getFormTitle()).eql('Set up YubiKey');
+    await t.expect(pageObject.getFormSubtitle()).eql('Insert the YubiKey into a USB port and tap it to generate a verification code.');
```

Comment:
Ohh, that is caused by the function trying to assert on console logs before the form renders, we have a solution for this, it is just to perform an assertion on the existence of the `form` element. This resolves the issue, but no worries, since this also effectively does the same thing no reason to insert another useless assertion. Ty!